sssd_test_framework/utils/sssd.py

```diff
@@ -406,7 +407,12 @@ def config_apply(self, check_config: bool = True, debug_level: str | None = "0xf
         :type debug_level:  str | None, optional
         """
         cfg = self.__set_debug_level(debug_level)
-        service_user = self.config.get("sssd", "user", fallback="root")
+        if self.fs.exists("/etc/systemd/system/multi-user.target.wants/sssd.service"):
+            service_file = self.fs.read("/etc/systemd/system/multi-user.target.wants/sssd.service")
```

Comment:
`systemctl cat sssd.service` would be better, its just one call and it also considers overwrites

Reply:
This should be done by extension of https://pytest-mh.readthedocs.io/en/latest/api/pytest_mh.utils.services.html#pytest_mh.utils.services.SystemdServices ?

Reply:
Not necessarily, but it would be nicer.

Reply:
https://github.com/next-actions/pytest-mh/pull/40

Reply:
Merged.